multiple_generate.sh

```diff
@@ -7,19 +7,18 @@ function clean() {
 function cloner() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  git clone $1 ${REPONAME}_temp &
+  git clone $1 ${REPONAME} &
 }
 
-function repoCleaner() {
+function genSubIndex() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  cd ${REPONAME}_temp
-  mkdir ../$REPONAME
-  cp -a docs/. ../$REPONAME/.
+  cd $REPONAME
+  python $BASE/modules/scripts/genindex.py ./
```

Comment:
why are you running `genindex.py` from the root of the sub-project? genindex should be run from `DOCPATH`. Last I checked, `docs` was the default DOCPATH, not `.`